pixeltable/exec/sql_scan_node.py

```diff
@@ -192,7 +203,7 @@ def __next__(self) -> DataRowBatch:
             if needs_row:
                 output_row = output_batch.add_row()
             if self.num_pk_cols > 0:
-                output_row.set_pk(tuple(sql_row[-self.num_pk_cols:]))
+                output_row.set_pk(tuple(sql_row[-self.num_pk_cols :]))
```

Comment:
This is completely non-standard formatting.

Reply:
Actually I don't think that's true. In compound expressions within a slice, it's normal to put a space between the expression and the colon.

See: https://peps.python.org/pep-0008/

In particular:

"In an extended slice, both colons must have the same amount of spacing applied. Exception: when a slice parameter is omitted, the space is omitted".

I believe the formatter is doing the correct thing in this case.

Reply:
Here's an example straight out of PEP8:

`ham[:: step_fn(x)]`